src/main/java/org/springframework/integration/dsl/IntegrationFlowAdapter.java

```diff
@@ -54,14 +55,47 @@
  *
  * @since 1.1
  */
-public abstract class IntegrationFlowAdapter implements IntegrationFlow {
+public abstract class IntegrationFlowAdapter implements IntegrationFlow, SmartLifecycle {
+
+	private StandardIntegrationFlow targetIntegrationFlow;
 
 	@Override
 	public final void configure(IntegrationFlowDefinition<?> flow) {
 		IntegrationFlowDefinition<?> targetFlow = buildFlow();
 		Assert.state(targetFlow != null, "the 'buildFlow()' must not return null");
 		flow.integrationComponents.clear();
 		flow.integrationComponents.addAll(targetFlow.integrationComponents);
+		this.targetIntegrationFlow = flow.get();
+	}
+
+	@Override
+	public void start() {
+		this.targetIntegrationFlow.start();
```

Comment:
Yep, it is!
Although that is the problem only if we don't use Spring Integration properly: missed `@EnableIntegration`, `new ...()` instead of `BeanFactory` etc...
Np. Let me fix it with an appropriate `Assert`!

The Travis throws `StackOverflow` somehow after these changes :cry: